Classes/Service/BackupService.php

```diff
@@ -148,13 +148,23 @@ public function createBackup()
         // create archive
         $compressor = $this->getCompressor();
         $archivePath = $compressor->compress($backupPath, $this->config['tempPath']);
+
+        if (!is_file($archivePath)) {
+            $this->logger->error('couldn\`t create backup archive file: ' . $archivePath);
+
+            return false;
```

Comment:
Please add a bool return type to this method.

https://github.com/breadlesscode/neos-backups/blob/87e01a21f63b375d68177ccd1c11621493021903/Classes/Service/BackupService.php#L134

Reply:
Yes, I've added it and made some small celanups